alcs-frontend/src/app/services/notice-of-intent/notice-of-intent.service.ts

```diff
@@ -56,7 +56,7 @@ export class NoticeOfIntentService {
 
   async fetchByFileNumber(fileNumber: string) {
     try {
-      return await firstValueFrom(this.http.get<NoticeOfIntentDto>(`${this.url}/${fileNumber}`));
+      return firstValueFrom(this.http.get<NoticeOfIntentDto>(`${this.url}/${fileNumber}`));
```

Comment:
I think you still want to have await here for better call stack

Reply:
Change reverted